.github/workflows/_buildpacks-release-detect.yml

```diff
@@ -8,9 +8,6 @@ on:
         description: The list of {buildpack_id, buildpack_version, buildpack_dir, buildpack_artifact_prefix, docker_repository} values for a buildpack formatted as a JSON array
         value: ${{ jobs.release-detect.outputs.buildpacks }}
 
-permissions:
-  contents: read
```

Comment:
I presume the `actions/checkout` action doesn't need `read` permissions since the repos are public?

I'm pretty sure removing this should work - worst case we can add it back later :-)

Reply:
from my testing it didn't appear to be required but there may be subtle differences between my personal org i used for testing and the heroku org that might not immediately surface (e.g.; IP whitelisting). 

one other thing i found was that permissions set in the shared workflow don't actually matter except in that they sometimes help display a more informative error message (i.e.; something like "workflow X requested read permission but it was not given"). 

only the primary workflow that calls the shared workflows controls the permission set (which makes sense) so this change shouldn't actually affect anything except that error message.